direct/nn/iterdualnet/iterdualnet_engine.py

```diff
@@ -0,0 +1,44 @@
+# coding=utf-8
+# Copyright (c) DIRECT Contributors
+
+from typing import Any, Callable, Dict, Optional, Tuple
+
+import torch
+from torch import nn
+
+from direct.config import BaseConfig
+from direct.nn.mri_models import MRIModelEngine
+
+
+class IterDualNetEngine(MRIModelEngine):
+    def __init__(
+        self,
+        cfg: BaseConfig,
+        model: nn.Module,
+        device: str,
+        forward_operator: Optional[Callable] = None,
+        backward_operator: Optional[Callable] = None,
```

Comment:
How come these are optional? Can you add a docstring

Reply:
They shouldn't be. Will fix